bftengine/src/preprocessor/PreProcessor.hpp

```diff
@@ -68,7 +68,8 @@ class PreProcessor {
   template <typename T>
   void onMessage(T *msg);
 
-  void registerClientPreProcessRequest(ClientPreProcessReqMsgUniquePtr clientReqMsg);
+  void registerRequest(ClientPreProcessReqMsgUniquePtr clientReqMsg,
+                       PreProcessRequestMsgSharedPtr preProcessRequestMsg);
```

Comment:
Yulia, what's the difference between `clientReqMsg` and `preProcessRequestMsg` ?

Reply:
PreProcessRequestMsg is what we send to non-primary replicas to ask for pre-execution.